definitions/npm/react-helmet_v6.x.x/flow_v0.104.x-/react-helmet_v6.x.x.js

```diff
@@ -0,0 +1,66 @@
+declare module 'react-helmet' {
+  declare interface OtherElementAttributes {
+    [key: string]: string | number | boolean | null | void;
+  }
+  declare interface HelmetTags {
+    baseTag: any[];
+    linkTags: HTMLLinkElement[];
+    metaTags: HTMLMetaElement[];
+    noscriptTags: any[];
+    scriptTags: HTMLScriptElement[];
+    styleTags: HTMLStyleElement[];
+  }
+  declare interface HelmetProps {
+    async?: boolean | void;
+    base?: any;
+    defaultTitle?: string | void;
+    defer?: boolean | void;
+    encodeSpecialCharacters?: boolean | void;
+    onChangeClientState?:
+      | ((
+          newState: any,
+          addedTags: HelmetTags,
+          removedTags: HelmetTags
+        ) => void)
+      | void;
+    noscript?: any[] | void;
+    script?: any[] | void;
+    style?: any[] | void;
+    title?: string | void;
+    titleAttributes?: { [key: string]: any } | void;
+    titleTemplate?: string | void;
+  }
+  declare class Helmet mixins React$Component<HelmetProps> {
+    static rewind(): HelmetData;
+    static renderStatic(): HelmetData;
+    static canUseDOM: boolean;
+  }
+  declare var HelmetExport: typeof Helmet;
+  declare export { HelmetExport as Helmet };
+  declare export default typeof HelmetExport;
```

Comment:
Can we export like this instead? Otherwise there are compatibility issues with require statements
```suggestion
  declare module.exports: typeof HelmetExport;
```

Reply:
@Brianzchen Better now?